src/client/src/index.css

```diff
@@ -23,8 +21,22 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 
+
+body{
+	height: 100vh;
```

Comment:
If we fix the height I think it makes sense to  make fixed all the heights, not %

Reply:
The difference is we need the body to fill all the view port, that why I used (100 View-Height) and the other elements in the app to be re-sized relative to their parents (50% for example)